frontend/package.json

```diff
@@ -76,21 +75,25 @@
     "@storybook/addon-links": "7.6.6",
     "@storybook/addon-onboarding": "1.0.10",
     "@storybook/blocks": "7.6.6",
+    "@storybook/builder-vite": "^7.6.6",
     "@storybook/preset-create-react-app": "7.6.6",
     "@storybook/react": "7.6.6",
     "@storybook/react-webpack5": "7.6.6",
     "@storybook/testing-library": "0.2.2",
-    "@testing-library/jest-dom": "^6.1.5",
     "@testing-library/react": "^14.1.2",
     "@testing-library/user-event": "^14.5.1",
+    "@vitest/coverage-v8": "^1.2.1",
```

Comment:
This would be good to move to devDependencies, along with other testing dependencies.

Reply:
I think it is there already ;-)